internal/provider/schema_cloudflare_list.go

```diff
@@ -0,0 +1,97 @@
+package provider
+
+import (
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceCloudflareListSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"account_id": {
+			Description: "The account identifier to target for the resource.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"name": {
```

Comment:
can you please add `Description`s to all of these new schema fields so that we can auto generate the documentation? see #1695 for an example of implementing this. once you're done, `make docs` will generate the output for you.